main/management/commands/kafka_consumer.py

```diff
@@ -6,11 +6,32 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
-from druncschema.broadcast_pb2 import BroadcastMessage
+from druncschema.broadcast_pb2 import BroadcastMessage, BroadcastType
 from kafka import KafkaConsumer
 
 from ...models import DruncMessage
 
+BROADCAST_TYPE_SEVERITY = {
+    BroadcastType.ACK: "INFO",
```

Comment:
One of the stated use cases was that they'd like to be able to filter out the constant ps messages that are sent so I think this does need to be debug and let's try out making `COMMAND_EXECUTION_START` debug as well.